### ISSUE TITLE
Field::media Doesn't display existing allowed media

### ISSUE THREAD

**Issue by putrefaction** [themosis/themosis#141]
I have this field : 

```javascript
Field::media('file', [
    'title'     => 'Fichier associé',
    'info'      => 'Some informations ...',
    'type'      => ['image', 'application']
  ])
```

In the administration, i can upload a PDF to attach it, but i can't see or select existing PDF files.
**WP 4.7 and Themosis 1.2**

**Comment by jlambe** [comment]
Yup it's normal, you need the Themosis framework version 1.3.0 at least in order to define multiple file "types" for a media or collection field. Normally, with your code you should only be able to select images.

You need to update to latest version but it has code refactoring. Please make sure you have the time to update for your project if the deadline is not too soon.
